### PR TITLE
ENH: Added entry for University of Belgrade

### LINKED ISSUE #33: Institute of Physics, Belgrade, Serbia

### PR DESCRIPTION
**Pull request by eaculb**
Closes #33

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -285,3 +285,11 @@ utoronto:
   state: Ontario
   street: 27 King's College Cir
   zip: M5S 3H7
+ubelgrade:
+  aka: [University of Belgrade, Универзитет у Београду, Univerzitet u Beogradu]
+  city: Belgrade
+  country: Serbia
+  institutes: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
```
Comment (sbillinge) [inline db/institutions.yml]: this should be departments.  You can't change the schema (i.e., you can't change the key names.)

File: db/institutions.yml [hunk 2]
```diff
@@ -285,3 +285,11 @@ utoronto:
...
+  name: University of Belgrade
```
Comment (sbillinge) [inline db/institutions.yml]: can you find a street address?  If so, please add.

**Comment by sbanerjee23** [comment]
@eaculb You're merging upstream master into your branch, not rebasing your branch into master. Don't use the github "Resolve Conflicts" thing on the webpage (pretty sure that's what you're doing). I messed this up the same way last week. Please test the command line rebasing instructions I added to the PR here (https://github.com/Billingegroup/group_git_workflow/pull/10), and let me or @chiahaoliu know if there are any issues. You're going to have to revert a commit before your merge attempt. This will be additionally complicated because you did it on the GitHub webpage, so it's not in your local history. I couldn't figure out how to revert properly, so eventually I ended up closing the PR, and submitted a new one. But there might be a better solution. Tim might know.

**Comment by eaculb** [comment]
Fixed it -- thanks Soham & Tim!

File: db/institutions.yml [hunk 3]
```diff
@@ -424,7 +424,8 @@ ubelgrade:
   aka: [University of Belgrade, Универзитет у Београду, Univerzitet u Beogradu]
   city: Belgrade
   country: Serbia
-  institutes: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
+  departments: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
   name: University of Belgrade
-  state: ''
+  street: Studentski trg 1
+  state: Beograd
```
Comment (sbillinge) [inline db/institutions.yml]: Please check.  Is Beograd the city (i.e., Belgrade in Serbian)?

Comment (eaculb) [reply db/institutions.yml]: Ah, you're right, it is. I'll remove the state line.

**Pull request merged**